.gitignore

```diff
@@ -8,4 +8,4 @@ build/
 .vscode/
 *.db
 coverage.info
-pistache/
\ No newline at end of file
+pistache/
```

Comment:
I assume we have gitignored this directory because the directory exists. Do you know how and what creates this directory.

Reply:
Hmmm are you sure it does? Can you send a link?

Reply:
Perhaps I didn't make myself clear. I was actually asking a question about the source of  `pistache/` directory. How is the directory produced, and what does it contain?

Reply:
Oh, now I understand. If you have a look at our [dependencies](https://github.com/platisd/indoor-navigation-system/blob/dev/get_dependencies.sh#L10) script you will see that it clones `pistache` in the root folder of the repository. Therefore if someone were to use the script for quickly fetching the dependencies (like our CI does) then they could accidentally commit `pistache` as well.